cEpiabm/src/dataclasses/person.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef EPIABM_DATACLASSES_PERSON_HPP
+#define EPIABM_DATACLASSES_PERSON_HPP
+
+#include <vector>
+#include <memory>
+
+namespace epiabm
+{
+
+    struct PersonParams
+    {
+        unsigned char age = 0;
+        float susceptibility = 0, infectiousness = 0;
+    };
+
+    enum class InfectionStatus
+    {
+        Susceptible,
+        Exposed,
+        InfectASympt,
+        InfectMild,
+        InfectGP,
+        InfectHosp,
+        InfectICU,
+        InfectICURecov,
+        Recovered,
+        Dead
```

Comment:
The E, R and D currently miss from the python implementation. Should we also add them there?